vingo/main.go

```diff
@@ -81,7 +89,28 @@ func setupFromEnv() {
 	}
 	corsAllowOrigins = cors_allow_origins
 
+	dev, dev_ok := os.LookupEnv("DEVELOPMENT")
+	if !dev_ok {
+		log.Fatal("DEVELOPMENT environment variable not set")
+	}
+	dev_value, dev_value_err := strconv.ParseBool(dev)
+	if dev_value_err != nil {
+		log.Fatal("DEVELOPMENT environment variable is not a valid boolean")
+	}
+	development = dev_value
+
 	// stuff for Zauth oauth flow
+
+	zauth_url, url_ok := os.LookupEnv("ZAUTH_URL")
+	if !url_ok {
+		log.Fatal("ZAUTH_URL environment variable not set")
+	}
+
+	zauth_callback_path, callback_ok := os.LookupEnv("ZAUTH_CALLBACK_PATH")
+	if !callback_ok {
+		log.Fatal("ZAUTH_CALLBACK_PATH environment variable not set")
+	}
+
```

Comment:
I think it makes more sense to somehow get this from fiber, since this is dependent on the path set in the code, not on anything external?

Reply:
It also depends on the path set in zauth (which needs to be adjusted when this gets merged)
We already had deployment issues before when this was hardcoded so if anything it'd be better to change the fiber path to the env variable.

Reply:
Nevermind setting the fiber path to the env variable won't work.
Still convinced it's better to keep it as an env variable though